js-pkg/tests/src/driftdb.test.ts

```diff
@@ -33,7 +33,7 @@ class CallbackExpecter<T> {
 
     accept = (value: T) => {
         if (this.timeout) {
-            clearTimeout(this.timeout);
+            clearTimeout(this.timeout as any as number);
```

Comment:
oh weird, is this a bun issue?

Reply:
seems to be :( but this makes it work

Reply:
actually, I think it's a `bun-types` issue rather than a bun issue